### DISCUSSION TITLE
能不能添加关键字屏蔽视频功能

### DISCUSSION THREAD

**Discussion by yebei199** [BewlyBewly/BewlyBewly#697] [Ideas]
这是一个非常非常实用, 并且强大的功能, 因为b站总是给你一直推荐不喜欢的视频, 点了不感兴趣也没有用, 如果能够直接通过关键字(视频标题和视频标签)屏蔽就太爽了,  如果有这个功能, 真是爽爽又歪歪啊

**Comment by TC999** [comment]
> 这是一个非常非常实用, 并且强大的功能, 因为b站总是给你一直推荐不喜欢的视频, 点了不感兴趣也没有用, 如果能够直接通过关键字(视频标题和视频标签)屏蔽就太爽了, 如果有这个功能, 真是爽爽又歪歪啊

[GreasyFork](https://greasyfork.org/zh-CN)有很多现成的脚本，一搜一堆

**Reply by yebei199** [reply]
和这个不兼容, 因为这个是直接用shadowdom覆盖的, 那些脚本无法识别这种dom

**Reply by hakadao** [reply]
已經根據播放量與時長過濾，後續加關鍵字或標籤
